helm/kubernetes-elastic-stack-elastic-logging/requirements.lock

```diff
@@ -1,18 +1,18 @@
 dependencies:
 - name: elasticsearch
   repository: https://giantswarm.github.com/sample-catalog
-  version: 0.1.7
+  version: 0.2.1
```

Comment:
Are these in the sample catalog? If so I think we should move them.

@pipo02mix WDYT?

Reply:
yeah think so too

Reply:
Where do I have to move them?

Reply:
They should also be in the incubator catalog.

Reply:
There is an error with the architect that is not allowing me to publish the "subcharts" to the incubator catalog https://github.com/giantswarm/giantswarm/issues/7271 could we move forward with this so customer can try out the chart?

Reply:
Yes no need to delay this.

Let's move the subcharts to the incubator as a later step. We should also rename the app to `elastic-stack-app` to match the new naming convention.